packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -117,61 +111,67 @@ export const makeInnerVault = (
   zcf,
   manager,
   assetNotifier,
-  idInManager, // will go in state
+  idInManager,
   runMint,
   priceAuthority,
 ) => {
   // CONSTANTS
   const collateralBrand = manager.getCollateralBrand();
   const { brand: runBrand } = runMint.getIssuerRecord();
 
-  // STATE
-
-  // #region Phase state
-  /** @type {InnerPhase} */
-  let phase = VaultPhase.ACTIVE;
+  /**
+   * State object to support virtualization when available
+   *
+   * @type {ImmutableState & MutableState}
+   */
+  const state = {
+    assetNotifier,
+    idInManager,
+    manager,
+    outerUpdater: null,
+    phase: VaultPhase.ACTIVE,
+    priceAuthority,
+    runMint,
+
+    // vaultSeat will hold the collateral until the loan is retired. The
+    // payout from it will be handed to the user: if the vault dies early
+    // (because the vaultFactory vat died), they'll get all their
+    // collateral back. If that happens, the issuer for the RUN will be dead,
+    // so their loan will be worthless.
+    vaultSeat: zcf.makeEmptySeatKit().zcfSeat,
+
+    // Two values from the same moment
+    interestSnapshot: manager.getCompoundedInterest(),
+    // @ts-expect-error until https://github.com/Agoric/agoric-sdk/pull/4736
+    runSnapshot: AmountMath.makeEmpty(runBrand),
+  };
 
+  // #region Phase logic
   /**
    * @param {InnerPhase} newPhase
    */
   const assignPhase = newPhase => {
+    const { phase } = state;
     const validNewPhases = validTransitions[phase];
     if (!validNewPhases.includes(newPhase))
```

Comment:
MUST add braces

Reply:
How did this get through review in the first place? We require braces for control constructs.

Reply:
This should just be an `assert`

Reply:
> How did this get through review in the first place? We require braces for control constructs.

I suppose because "require braces for control constructs" is not in https://github.com/Agoric/agoric-sdk/wiki/Coding-Style or the lint configuration. IMO if something is a rule and can be reliably checked by the machine, then humans shouldn't pay attention to looking for it. https://github.com/Agoric/agoric-sdk/issues/4747

In this case I'll change to `assert`